src/plugins/intel_cpu/src/nodes/reshape.cpp

```diff
@@ -81,6 +78,9 @@ bool Reshape::needShapeInfer() const {
             return true;
         }
     }
+    if (inputShapesModified()) {
+        return true;
+    }
     return false;
```

Comment:
```sugestion
    return inputShapesModified();
```
Maybe when there will be opportunity to update.

Reply:
sorry, I didn't get it, do you mean below suggestion?
```suggestion
   return inputShapesModified();
```